### ISSUE TITLE
FR: Transfer current playlist

### ISSUE THREAD

**Issue by gerroon** [fatg3erman/RompR#28]
Hi

This is a feature that I stealing from Yatse Android app. 

In Yatse if you change the player, Yatse asks you if you also want to move the current playlist to the new player therefor, lets you resume your playlist in the new player. I feel like this can be beneficial to the users even if they are using a multiroom setup or not. For instance the user can keep the playlist for the player set for mobile streaming etc.

**Comment by fatg3erman** [comment]
I could do this for multiple Players in Rompr, provided they are all Mopidy, or all MPD.

**Comment by gerroon** [comment]
Yeah that makes sense. Naturally the songs have to be playable by the players. Anyway thanks for considering.
